skorch/net.py

```diff
@@ -493,7 +493,7 @@ def train_step(self, Xi, yi, **fit_params):
         y_pred = self.infer(Xi, **fit_params)
         loss = self.get_loss(y_pred, yi, X=Xi, training=True)
         loss.backward()
-        self.notify('on_grad_computed', parameters=self.module_.parameters())
```

Comment:
Should the argument not also be named `named_parameters` then?

Reply:
Done